src/main/java/uk/gov/hmcts/reform/cwrdapi/service/CaseWorkerProfileConverter.java

```diff
@@ -117,7 +117,7 @@ private boolean isSuspended(CaseWorkerProfile obj) {
         return caseWorkerLocationRequests;
     }
 
-    private Set<String> generateIdamRoles(String idamRoles) {
+    public Set<String> generateIdamRoles(String idamRoles) {
```

Comment:
Any reason for changing it to public?

Reply:
Makes it easier to directly test the method and cover any mutations - can revert if necessary

Reply:
Reverted here: https://github.com/hmcts/rd-caseworker-ref-api/pull/240/commits/b26a96500916bb6ad0b68f3869d4bf415045d440